pkg/cmd/server/origin/auth_config.go

```diff
@@ -26,6 +26,10 @@ type AuthConfig struct {
 	AssetPublicAddresses []string
 	EtcdHelper           storage.Interface
```

Comment:
Heh, I guess I should have been specific.  This needs to be doc'ed too to explain the difference between it and `EtcdBackends`.  When other people look, it should be obvious which they should use.